terracumber_config/tf_files/SUSEManager-Head-NUE.tf

```diff
@@ -12,12 +12,12 @@ variable "CUCUMBER_COMMAND" {
 
 variable "CUCUMBER_GITREPO" {
   type = string
-  default = "https://github.com/uyuni-project/uyuni.git"
+  default = "https://github.com/SUSE/spacewalk.git"
```

Comment:
Keep using uyuni repo please, we don't need to touch Manager-4.2, we don't need to port it to spacewalk repo.

Reply:
FYI, once 4.3 GM is ready, Julio will create a branch in spacewalk repo Manager-4.3 with will be a copy of the latest master changes in uyuni repo. That's our procedure for every new version.

Reply:
Ok but the branch I worked with was created in spacewalk: https://github.com/SUSE/spacewalk/pull/17629

Reply:
That's wrong sorry :(
Notice my warning in the description "We only want to do it in Head/4.3/Uyuni products."

Reply:
Fixed.